src/Blocks/BlockButton.h

```diff
@@ -67,7 +67,7 @@ class cBlockButtonHandler :
 			case BLOCK_FACE_XM: return 0x2;
 			case BLOCK_FACE_XP: return 0x1;
 			case BLOCK_FACE_YM: return 0x0;
-			default:
+			case BLOCK_FACE_NONE:
```

Comment:
Is this for all enums? It might get a bit verbose if someone switch'd MonsterType and ended up having to type out all the values that default handled.

If default can still be used, excuse me for misunderstanding.